### ISSUE TITLE
The app failed. The application has eliminated my new scenario.

### ISSUE THREAD

**Issue by Alabulie1978** [NPBruce/valkyrie#593]
Good evening, I apologize for the translation, my English is very basic and I'm going to use an online translator.

I have a problem with the app. During this week I created a scenario and this afternoon I finished it, but the app failed. I checked the stage to see that it worked correctly.

The problem is that the app prevents me from starting the scene again and does not allow me to edit again. Press the edit button and send me to the start menu, press the start button does not advance anything, remains on the first screen.

I send you my problem because I would like to play it and share it with the community. What can be the problem and I would like to know if I can solve it or I have definitely lost the scenario.

**Comment by Hobbeslionheart** [comment]
I have had that happen to me at least a few times. I tend to go back in the files in the folder that NPBruce point to and delete the most recent events or things I made. I found that UI have given me trouble and broke my scenario. So I haven't really bothered with them to date although I should give them a try again soon.

**Comment by NPBruce** [comment]
There are no known bugs in the editor that do this, so I would like to know if you have a reproducible case or if I get a copy of the broken scenario I can try to work out how this can happen.

**Comment by Alabulie1978** [comment]
Good afternoon bruce and thanks for your response, check the address you indicated to me, did not know the storage location of the scenarios and I thank you, I thought I had lost the whole stage.

Now I tell you what I have discovered. I have managed to repair the scenario by removing part of the code and now it works again. I did not understand what the problem was, I had created a creature, it was a zombie, but I needed a zombie horde, so I created a mob and added features of undead, this caused the stage to stop working. When you remove the creature, it will work again.

I hope it will help you to see if there are any bugg


PS: I'm finishing my first stage in Spanish, when I check it and see what works well I'll upload it so the community can enjoy it too
